src/arch/xtensa/smp/idc.c

```diff
@@ -69,7 +69,7 @@ static void idc_irq_handler(void *arg)
 			idc->received_msg.extension =
 					idctefc & IPC_IDCTEFC_MSG_MASK;
 
-			schedule_task(&idc->idc_task, 0, IDC_DEADLINE);
+			schedule_sof_task(&idc->idc_task, 0, IDC_DEADLINE, 0);
```

Comment:
no need for _sof_ in the function naming here either. Removing this helps with the 80 char line length.

Reply:
Done